packages/expo-in-app-purchases/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "expo-in-app-purchases",
+  "version": "1.0.0",
+  "description": "ExpoInAppPurchases standalone module",
+  "main": "build/InAppPurchases.js",
+  "types": "build/InAppPurchases.d.ts",
+   "scripts": {
+    "build": "expo-module build",
+    "clean": "expo-module clean",
+    "test": "expo-module test",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepublishOnly",
+    "expo-module": "expo-module"
+  },
+  "keywords": [
+    "react-native",
+    "expo",
+    "expo-in-app-purchases"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/expo/expo.git"
+  },
+  "bugs": {
+    "url": "https://github.com/expo/expo/issues"
+  },
+  "author": "650 Industries, Inc.",
+  "license": "MIT",
+  "homepage": "https://github.com/expo/expo/tree/master/packages/expo-in-app-purchases",
+  "peerDependencies": {
+    "@unimodules/core": "~1.0.0"
+  },
+  "gitHead": "8413e821076a2eca36b302e68dd628ce2e1f591a"
+
+,"_resolved": ""
+,"_integrity": ""
+,"_from": "expo-in-app-purchases@latest"
```

Comment:
These three props are leftovers from `expo generate-module`, you could remove them safely